pilot/pkg/networking/core/v1alpha3/listener_builder.go

```diff
@@ -254,6 +236,9 @@ func (builder *ListenerBuilder) buildManagementListeners(_ *ConfigGeneratorImpl,
 	return builder
 }
 
+// This function builds the virtual outbound listener in pretty much the same way
+// as a standard outbound listener that has HTTP sniffing enabled.
+// In addition, it adds a filter chain to prevent traffic loops when a pod calls itself using its IP.
```

Comment:
You donot need this as currently iptables donot intercept this kind of traffic

Reply:
What do you mean? We have a feature for restrictpodiptrafficloop that adds a filter chain match. Is that no longer valid?

Reply:
The traffic of app calling itself by podip does not flow through envoy now. Since the pr i did.

Reply:
#19308

Reply:
So https://github.com/istio/istio/pull/21332/files#diff-45fabefc3a80af1fc0ed5ebcf0b32dfeR257 block is redundant?